core/domain/learner_progress_services_test.py

```diff
@@ -465,6 +469,28 @@ def test_remove_collection_from_incomplete_list(self):
         self.assertEqual(self._get_all_incomplete_collection_ids(
             self.user_id), [])
 
+
```

Comment:
Remove unwanted newlines.

Reply:
Ah right. I forgot to delete those. I did that to make it more structured for myself.